@xen-orchestra/proxy/src/app/mixins/api.js

```diff
@@ -70,6 +70,13 @@ export default class Api {
         return
       }
 
+      if (typeof result.pipe === 'function') {
```

Comment:
Take a look at the `isAsyncIterable` test, you missed something.

Reply:
```suggestion
      if (typeof result?.pipe === 'function') {
```